docs/kb/semgrep-code/semgrep-scan-troubleshooting.md

```diff
@@ -10,7 +10,7 @@ description: Troubleshoot common issues with Semgrep scans.
 If a Semgrep scan is failing or running slowly, try the following steps to investigate:
 
 1. [Update Semgrep](/docs/update) to the latest version, if you are not currently running the latest version. Some errors result from an older version of Semgrep being used with newer rules.
-2. Re-run the scan with either the `-v`/`--verbose` or `--debug` (extremely verbose) flags. These options provide more information about what is failing.
+2. Re-run the scan with either the `-v`/`--verbose` or `--debug` (extremely verbose) flags. These options provide more information about what is failing. Keep in mind, to get the desired output, you will also likely have to set `--max-log-list-entries` to `0` (or any negative value) as it is set to a max of `100` by default and you may have more entries in your log output than that.
```

Comment:
```suggestion
2. Re-run the scan with either the `-v`/`--verbose` or `--debug` (extremely verbose) flags. These options provide more information about what is failing. 
  * When using verbose logs, also set `--max-log-list-entries` to `0` or any negative value to see the full output; otherwise lists of rules and files suppressed if there are more than 100 entries in the list.
```

Reply:
Missing a word in the suggestion?
> ...lists of rules and files **are** suppressed...